scripts/sct_straighten_spinalcord.py

```diff
@@ -15,235 +15,242 @@
+        help='Acceleration factor for the calculation of the straightening warping field.'
+             ' This speed factor enables an intermediate resampling to a lower resolution, which '
+             'decreases the computational time at the cost of lower accuracy.'
+             ' A speed factor of 2 means that the input image will be downsampled by a factor 2 '
+             'before calculating the straightening warping field. For example, a 1x1x1 mm^3 image '
+             'will be downsampled to 2x2x2 mm3, providing a speed factor of approximately 8.'
+             ' Note that accelerating the straightening process reduces the precision of the '
+             'algorithm, and induces undesirable edges effects. Default=1 (no downsampling).',
+        required=False,
+        default=1)
+    optional.add_argument(
+        "-xy-size",
+        metavar=Metavar.float,
+        type=float,
+        help='Size of the output FOV in the RL/AP plane, in mm. The resolution of the destination '
+             'image is the same as that of the source image (-i).',
+        required=False,
+        default=35.0)
+    optional.add_argument(
+        "-o",
+        metavar=Metavar.file,
+        help="Straightened file. Example: data_straight.nii.gz",
+        required = False,
+        default = '')
+    optional.add_argument(
+        "-ofolder",
+        metavar=Metavar.folder,
+        help="Output folder (all outputs will go there).",
+        required=False,
+        default='./')
+    optional.add_argument(
+        '-centerline-algo',
+        help='Algorithm for centerline fitting.',
+        choices = ('bspline', 'linear', 'nurbs'),
+        default = 'nurbs')
+    optional.add_argument(
+        '-centerline-smooth',
+        metavar=Metavar.int,
+        type=int,
+        help='Degree of smoothing for centerline fitting. Only use with -centerline-algo {bspline, linear}.',
+        default=10)
 
-    parser.add_option(name='-centerline-algo',
-                      type_value='multiple_choice',
-                      description='Algorithm for centerline fitting.',
-                      mandatory=False,
-                      example=['bspline', 'linear', 'nurbs'],
-                      default_value='nurbs')
-    parser.add_option(name='-centerline-smooth',
-                      type_value='int',
-                      description='Degree of smoothing for centerline fitting. Only use with -centerline-algo {bspline, linear}.',
-                      mandatory=False,
-                      default_value=10)
+    optional.add_argument(
+        "-param",
+        metavar=Metavar.list,
+        help="R|Parameters for spinal cord straightening. Separate arguments with ','."
+             "\nprecision: [1.0,inf[. Precision factor of straightening, related to the number of slices. Increasing this parameter increases the precision along with increased computational time. Not taken into account with hanning fitting method. Default=2"
```

Comment:
i would add a space for visual purpose and to be consistent with previous refactoring

note: the same applies for every other similar case of course

Reply:
So every time there is an instance of separating numbers (example: 2,3,4) we would now apply space?